test/parallel/test-cluster-shared-handle-bind-privileged-port.js

```diff
@@ -35,6 +36,18 @@ if (common.isWindows)
 if (process.getuid() === 0)
   common.skip('as this test should not be run as `root`');
 
+if (common.isLinux) {
```

Comment:
More comments here would be helpful as it is not super obvious why this is needed.